portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSet.testcase

```diff
@@ -59,80 +59,6 @@ definition {
 		}
 	}
```

Comment:
Commit message says "Playwright"

Reply:
What do you mean?. 
Tests removed in this commit were kind of duplicated. They were covering the same cases as the `DataSetViewPagination.testcase` and those were migrated to PW.

Reply:
Oh I see. My interpretation was "Remove pagination tests covered in Poshi" because you just migrated to PW. But real meaning us "Remove pagination tests already covered in PW"

So far so good!